setup.ts

```diff
@@ -529,6 +544,16 @@ function validateRecaptcha(string: string): boolean {
   return pattern.test(string);
 }
 
+/**
+ * The function validates whether a given image size is less than 20 and greater than 0.
+ * @param string - The `number` parameter represents the input size of the string
+ * validated. In this case, it is expected to be a number less than 20 and greater than 0.
+ * @returns a boolean value.
+ */
+function validateImageFileSize(size: number): boolean {
+  return size > 0 && size <= 20;
```

Comment:
1. If it's greater than 20, just set the value to 20
2. Use a variable instead of 20 so that we can easily change it in future